azure-pipelines.yml

```diff
@@ -39,13 +39,17 @@ jobs:
       project: build
       pipeline: 9
       artifacts: sonic-swss-common
+      runVersion: 'latestFromBranch'
+      runBranch: 'refs/heads/master'
```

Comment:
>master [](start = 29, length = 6)

Not a blocking issue: is it possible to figure out the branch automatically, so it will work on any branches like 202012, etc.

Reply:
you are right.